src/cli.js

```diff
@@ -491,6 +509,35 @@ async function build() {
       return resolveConfig()
     }
 
+    let [beforePlugins, afterPlugins] = includePostCss ? await loadPostCssPlugins() : [[], []]
+
+    let plugins = [
+      ...beforePlugins,
+      '__TAILWIND_PLUGIN_POSITION__',
+      !args['--minify'] && formatNodes,
+      ...afterPlugins,
+      !args['--no-autoprefixer'] &&
+        (() => {
+          // Try to load a local `autoprefixer` version first
+          try {
+            return require('autoprefixer')
+          } catch {}
+
+          return lazyAutoprefixer()
+        })(),
+      args['--minify'] &&
+        (() => {
+          let options = { preset: ['default', { cssDeclarationSorter: false }] }
+
+          // Try to load a local `cssnano` version first
+          try {
+            return require('cssnano')(options)
```

Comment:
If there is a local cssnano should we not override the options? The only reason we override is because of the weird JSON file dependency thing which we can't figure out how to easily pre-bundle, but if cssnano is installed locally then I think that problem goes away.